index.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module "nice-try" {
+  export default function niceTry(f: Function): any;
```

Comment:
`any` will introduce unsafe typing holes - Consider something like:
```
const foo = niceTry(() => 'hello');
foo().doesNotExist()
```

Will throw at runtime, but TypeScript will say it's OK due to the `any`.

You probably want to use generics here instead.

Reply:
I just read the documentation and `ReturnType` looks promising in this case.

Reply:
Can you have a second look? 😄